tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -26,10 +26,9 @@ Developers must perform the following actions to see their resources on the dash
       system:
     ```
 
-1. Commit and push the Component definition to a Git repository that is registered as a Catalog Location. See [Adding
+2. Commit and push the Component definition to a Git repository that is registered as a Catalog Location. See [Adding
```

Comment:
What is a catalog? Can we link to a definition?

Reply:
'Adding Catalog entities' is already linked in the below bullet point (point #2)